src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -949,19 +949,27 @@ export class ChallengeModal extends Modal<Events, ChallengeModalProperties, any>
                         <label className="control-label" htmlFor="engine">
                             {pgettext("Computer opponent", "AI Player")}
                         </label>
-                        <div className="controls">
-                            <select
-                                id="challenge-ai"
-                                value={this.state.conf.bot_id}
-                                onChange={this.update_conf_bot_id}
-                                required={true}
-                            >
-                                {bots_list().map((bot, idx) => (
-                                    <option key={idx} value={bot.id}>
-                                        {bot.username} ({rankString(getUserRating(bot).rank)})
-                                    </option>
-                                ))}
-                            </select>
+                        <div>
+                            <div className="controls">
+                                <select
+                                    id="challenge-ai"
+                                    value={this.state.conf.bot_id}
+                                    onChange={this.update_conf_bot_id}
+                                    required={true}
+                                >
+                                    {bots_list().map((bot, idx) => (
+                                        <option key={idx} value={bot.id}>
+                                            {bot.username} ({rankString(getUserRating(bot).rank)})
+                                        </option>
+                                    ))}
+                                </select>
+                            </div>
+                            <div>
+                                <a href={`/user/view/${this.state.conf.bot_id}`}>
+                                    <PlayerIcon id={this.state.conf.bot_id} size={16} />{" "}
+                                    {bots()[this.state.conf.bot_id].username}'s profile
```

Comment:
Not sure if we want this spelling, but if we do, this should be translated.  Something like:

    interpolate(_("{{who}}'s profile"), {who: bots()[...]})

Reply:
Makes sense. I just now found the [wiki page on Internationalization and Localization](https://github.com/online-go/online-go.com/wiki/Internationalization-and-Localization). I was sort of expecting a mention of translation in CONTRIBUTING.md — that's where I looked and didn't see anything. Not sure if pointing to the wiki from the source is what should occur, but some pointer on translation in that file would've been useful for me.